bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/Root.scala

```diff
@@ -44,6 +44,6 @@ object Root {
 
   // REL-3769: GPI not available in 2020B.
   // GHOST not yet available
-  val choices: List[Instrument] = List(Alopeke, Flamingos2, /*Ghost,*/ GmosNorth, GmosSouth, Gnirs, /*Graces,*/ Gsaoi, Igrins, MaroonX, Nifs, /*Niri,*/ Visitor, Zorro)
```

Comment:
Maybe this needs to be rebased to get the latest changes

Reply:
As we have been talking about that change was not made by me although it may seem like it was. This was due to the process by which I made the merge. The solution is the one discussed with Carlos and would be to start from the HEAD of the development branch and from there include the ITC changes. I am going to make a new pull request following the steps mentioned above. 

e3da10496d (Carlos Quiroz 2023-02-21 16:00:25 -0300 47)   val choices: List[Instrument] = List(Alopeke, Flamingos2, /*Ghost,*/ GmosNorth, GmosSouth, Gnirs, /*Graces,*/ Gsaoi, Igrins, MaroonX, Nifs, /*Niri,*/ Visitor, Zorro)